src/System.Diagnostics.EventLog/tests/EventLogTests/EventLogTests.cs

```diff
@@ -26,15 +25,36 @@ public void CLearLogTest()
             EventLog myLog = new EventLog();
             myLog.Source = "MySource";
             myLog.WriteEntry("Writing to event log.");
+            Assert.Equal(1, myLog.Entries.Count);
             myLog.Clear();
             Assert.Equal(0, myLog.Entries.Count);
         }
 
-        [Fact]
+        [ConditionalFact(typeof(PlatformDetection), nameof(PlatformDetection.IsNotWindowsNanoServer))]
         public void ApplicationEventLog_Count()
         {
             EventLog ael = new EventLog("Application");
             Assert.InRange(ael.Entries.Count, 1, Int32.MaxValue);
         }
+
+        //[ConditionalFact(nameof(IsProcessElevated))]
+        [ConditionalFact(typeof(PlatformDetection), nameof(PlatformDetection.IsNotWindowsNanoServer))]
+        public void DelteLogTest()
```

Comment:
I may have already made this feedback, but if you chose a random name like this, you don't need to check it already exists. eg just
``` c#
string source = Guid.NewGuid().ToString("N");
string logName = Guid.NewGuid().ToString("N");
EventLog.CreateEventSource(source, logName);
Assert.True(EventLog.Exists(logName));
EventLog.Delete(logName);
Assert.False(EventLog.Exists(logName));
```

I iddn't use LogNameFromSourcename because then I'm testing two things in one test. that should be a separate test. As you can see, you end up with a lot of short focused tests.

Reply:
A purist would not have the Assert.True here because they would already have a test that verified that CreateEventSource led to Assert.True. And that would be fine.